astroquery/heasarc/core.py

```diff
@@ -1,7 +1,8 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 import warnings
-from six import BytesIO
+from six import StringIO, BytesIO
+from astropy.io.fits.column import _AsciiColumnFormat
```

Comment:
we shouldn't use `_`-preceded tools - they are private and therefore are not guaranteed to be available